README.md

```diff
@@ -133,6 +133,12 @@ With that setup, you could do the following in the terminal:
 - **Initialize a new project:** `npx gulp init`
 - **Update USWDS static assets and recompile:** `npx gulp update`
 
+### Usage tips
+- **Use `init` only once.** The `init` task is meant for initializing the design system on a project. Since it will overwrite project files (like settings files and the Sass entry point), use it sparingly and don't use it for updating the design system on a project, or at any point after you've customized your settings fies.  
+- **Update USWDS assets with `copyAssets`.** Don't update assets with `init`, use the `copyAssets` task. This task updates static assets (like images, fonts, and compiled JavaScript) only and you don't risk clobbering your customizations.
+- **Compile only from a single Sass entry point.** Define the location of this entry point in `paths.dist.theme`. Point to any other Sass file you need from this single entry point. If you have project Sass files outside the `paths.dist.theme` directory that you'd like to watch, you can set this location with `paths.src.projectSass`, but the Sass will still compile from the single entry point located in `paths.dist.theme`.
+- **Only check theme files and custom icons into version control.** You should have a build process that copies static assets like images, fonts, and compiled JavaScript from the `uswds` package. This assures that these assets are up-to-date with whatever version of USWDS you're using. You only need to track your customizations (like settings, theme files, custom icons, and your Gulpfile) in version control.  
```

Comment:
```suggestion
- **Only check theme files and custom icons into version control.** You should have a build process that copies static assets like images, fonts, and compiled JavaScript from the `uswds` package. This assures that these assets are up-to-date with whatever version of USWDS you're using. You only need to track your customizations (like settings, theme files, custom icons, and your gulpfile) in version control.  
```

Looks like `gulpfile` is typically lowercase: https://gulpjs.com/docs/en/getting-started/javascript-and-gulpfiles/